src/app/services/content-management.service.spec.ts

```diff
@@ -1429,7 +1429,7 @@ describe('ContentManagementService', () => {
       tick();
       flush();
 
-      expect(contentManagementService.linksUnshared.next).toHaveBeenCalledWith(jasmine.any(Object));
+      expect(contentManagementService.linksUnshared.next).toHaveBeenCalledWith();
```

Comment:
`toHaveBeenCalled()` and remove `fit`

Reply:
My bad, slight oversight on my part. Fixed now